### PR TITLE
+ farmhash 0.4

### PR DESCRIPTION
**Pull request by rr0gi**
switch to dune

### CODE REVIEW

**Comment by rr0gi** [comment]
ci failures are due to depexts missing for some platforms

File: packages/farmhash/farmhash.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+authors: "Ahrefs <github@ahrefs.com>"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/ocaml-farmhash"
+dev-repo: "git+https://github.com/ahrefs/ocaml-farmhash.git"
+bug-reports: "https://github.com/ahrefs/ocaml-farmhash/issues"
+tags: ["org:ahrefs" "clib:farmhash"]
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "dune" {>= "2.0"}
```
Comment (mseri) [inline packages/farmhash/farmhash.0.4/opam:12]:
```suggestion
  "dune" {>= "2.0"}
  "conf-c++"
```

File: packages/farmhash/farmhash.0.4/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+]
+depexts: [
```
Comment (mseri) [inline packages/farmhash/farmhash.0.4/opam:14]: Can you remove the depexts and instead use `conf-c++`, which includes the depexts for a lot more systems? I would have done it myself, but I am not allowed to edit this PR

**Comment by rr0gi** [comment]
done, thanks for the suggestion

File: packages/farmhash/farmhash.0.4/opam [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+tags: ["org:ahrefs" "clib:farmhash"]
+build: [ "dune" "build" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "dune" {>= "2.0"}
+  "conf-c++"
+]
+depexts: [
+  ["libfarmhash-dev"] {os-family = "debian"}
+]
```
Comment (mseri) [inline packages/farmhash/farmhash.0.4/opam:17]: I could not find rpms or the packages for alpine or arch. They can be added later if somebody finds them

**Pull request merged**